frontend/src/pages/GranteeRecord/components/GrantsList.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Container from '../../../components/Container';
+import './GrantsList.css';
+
+export default function GrantsList({ summary }) {
+  const getGrantPrograms = (programs) => {
+    let shortProgramTypes = [];
+    if (programs && Array.isArray(programs) && programs.length > 0) {
```

Comment:
I think there may be more programs then just these three, but not 100% sure.

Reply:
Now that there is a program table (and it shows the abbreviated name) this will be refactored